### DISCUSSION TITLE
"Analog-Heavy" AVR EA will have one 10-bit DAC

### DISCUSSION THREAD

**Discussion by trimarco232** [SpenceKonde/DxCore#254] [Future development discussion]
[https://ww1.microchip.com/downloads/en/DeviceDoc/00004376.pdf](url)

**Comment by SpenceKonde** [comment]
Okay, we've known this for over a year, it's in the product brief, along with a lot more detail? It's already supported on DxCore for the Dx-series

Frankly, there's a jarring split in their roadmap - Which is their analog flagship? The one with more flash, faster CPU and **on-chip opamps,** but a not-really-differential ADC (Vadc(max) = Vref, see datasheet - which bloody well defeats the point of a differential ADC). Or is it the one with the 12-bit proper differential ADC with the PGA? Both have a DAC, and a comparable ratio of analog comparators to pins. I have one prediction for how they set the EA apart from the tinyAVR, but I'll keep it to myself - thus far, only pincount and a dac distinguish them, 

I think the thing that we should be eying nervously is the bit about RWW, and what that might mean for the programming procedure - it likely means another version of NVMCTRL.
